### PR TITLE
Adds Chili con Carnival

### PR DESCRIPTION
**Pull request by TheVekter**
## About The Pull Request

I had like five minutes to kill and someone in the small suggestions thread said this would be funny.

image

Sprite by @ArcaneMusic 

## Why It's Good For The Game

Anything that involves stealing the Clown's shoes is good.

## Changelog
:cl:
add: Added Chili con Carnival, a new chili dish that requires the clown's shoes to make! (Sprite by ArcaneMusic)
/:cl:

### CODE REVIEW

**Comment by Farquaar** [comment]
The purple chili kind of looks like a floppy clown shoe sticking out of the bowl.

**Comment by TheVekter** [comment]
> The purple chili kind of looks like a floppy clown shoe sticking out of the bowl.

Hey, cool, my lazy ass coder sprite actually works.

File: code/modules/food_and_drinks/food/snacks_soup.dm [hunk 1]
```diff
@@ -124,6 +124,15 @@
 	tastes = list("tomato" = 1, "mint" = 1)
 	foodtype = VEGETABLES | MEAT
 
+/obj/item/reagent_containers/food/snacks/soup/clownchili
+	name = "chili con carnival"
+	desc = "A delicious stew of meat, chiles, and salty, salty clown tears."
+	icon_state = "clownchili"
+	bonus_reagents = list(/datum/reagent/consumable/nutriment = 1, /datum/reagent/consumable/tomatojuice = 2, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/consumable/laughter = 1)
+	list_reagents = list(/datum/reagent/consumable/nutriment = 5, /datum/reagent/consumable/capsaicin = 1, /datum/reagent/consumable/tomatojuice = 2, /datum/reagent/consumable/nutriment/vitamin = 2, /datum/reagent/consumable/banana = 1)
+	tastes = list("tomato" = 1, "hot peppers" = 2, "clown feet" = 2, "kind of funny" = 2, "someone's parents" = 2)
```
Comment (AnturK) [inline code/modules/food_and_drinks/food/snacks_soup.dm]: :thinking:

Comment (PKPenguin321) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: That clown had a family

Comment (TheVekter) [reply code/modules/food_and_drinks/food/snacks_soup.dm]: I changed this to say "a hint of childhood trauma".

**Comment by Krysonism** [comment]
I suggest considering making new soups based on my new bowl sprite rather than the old sprite. I understand if you prefer consistency with the two other chilis though.

bowlcomparasion

**Comment by TheVekter** [comment]
No, it's not a consistency thing, I just can't fucking sprite. If someone wants to make a sprite for this, by all means, My idea was to have the bowl be alternating colors of red, yellow, and blue, the chili be like... normal browns and reds with some yellow and purple bits floating throughout, then having a piece of a clown shoe sticking out like the spicy chile in the old sprite.

**Comment by TheVekter** [comment]
Lovely new sprite by @ArcaneMusic

**Pull request merged**